plugins/output/flight/sample.conf

```diff
@@ -0,0 +1,9 @@
+[[outputs.flight]]
+    ## Location to connect to
```

Comment:
If '#' and '##' have the same effect maybe use '#' everywhere for consistency.

Reply:
Fixed.